gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
I offer to use `6.2.2` version of gradle (`https\://services.gradle.org/distributions/gradle-6.2.2-all.zip`)

Reply:
If to be honest, I am not sure that it will correctly work. Please, check it

Reply:
But here it shows 3.6.0 as the latest plugin version and 5.6.4 as the latest Gradle version.
https://developer.android.com/studio/releases/gradle-plugin

Reply:
If `6.2.2` is available then I'll test if its building and update to that, otherwise I think we should make it `5.6.4`.

Reply:
> But here it shows 3.6.0 as the latest plugin version and 5.6.4 as the latest Gradle version.
> developer.android.com/studio/releases/gradle-plugin

Actually what this page indicated is that you need **at least** Gradle 5.6.4 if you plan to use 3.6.0.

Reply:
@tmortagne Yes, thanks for pointing out.